app/dashboard/dashboard.module.js

```diff
@@ -0,0 +1,4 @@
+import dashboard from './dashboard.component';
+
+export default angular.module('dashboard', [])
+    .component('dashboard', dashboard);
```

Comment:
use some prefix for your components for example pgDashboard. this is good approach to avoid name collisions with other directives